### PR TITLE
Adding OpenVDB to rhel for Galactic release of STVL

### PR DESCRIPTION
**Pull request by SteveMacenski**
Index: https://src.fedoraproject.org/rpms/openvdb

### CODE REVIEW

**Comment by SteveMacenski** [comment]
This is only in RHEL 8 (and probably future newer), should I add? 

```
  rhel:
    '7': null
    '*': [openvdb]
```

Curious - why is it checking for `7`? https://www.ros.org/reps/rep-2000.html REP 2000 only calls out RHEL 8

**Comment by cottsay** [comment]
> This is only in RHEL 8 (and probably future newer), should I add?

Yes, but please place the wildcard entry first in the list.

> Curious - why is it checking for `7`? https://www.ros.org/reps/rep-2000.html REP 2000 only calls out RHEL 8

We targeted RHEL 7 in ROS 2 prior to Galactic, and RHEL 7 still has a moderately large user base. REP 2000 really only captures declared support - it doesn't mean that folks don't use it on other platforms or releases. For now, ROS 2 is still usable on RHEL 7 using software collections, so there isn't much reason to drop it in rosdep.

One more fix: the `-dev` rule should use the `openvdb-devel` subpackage for RHEL. I'm sorry that src.fedoraproject.org doesn't list subpackages.

**Comment by SteveMacenski** [comment]
done!

File: rosdep/base.yaml [hunk 1]
```diff
@@ -4190,6 +4193,9 @@ libopenvdb-dev:
   fedora: [openvdb-devel]
   gentoo: [media-gfx/openvdb]
   nixos: [openvdb]
+  rhel:
+    '*': [openvdb-dev]
```
Comment (cottsay) [inline rosdep/base.yaml:4197]:
```suggestion
    '*': [openvdb-devel]
```

**Comment by SteveMacenski** [comment]
I think its good to go now, thanks for the help

**Review by cottsay — APPROVED** [review]
Thank you for taking the time to add the missing rule rather than just skipping the generator in Bloom.

**Comment by SteveMacenski** [comment]
No problemo, can we get this merged? I can then bloom up Galactic STVL

**Review by tfoote — APPROVED** [review]

**Pull request merged**